linkcheck/htmlutil/linkparse.py

```diff
@@ -175,10 +175,11 @@ def start_element (self, tag, attrs, element_text, lineno, column):
         log.debug(LOG_CHECK, "line %d col %d", lineno, column)
         if tag == "base" and not self.base_ref:
             self.base_ref = attrs.get("href", u'')
-        if tag =="a" and attrs.get_true('class', u''):
-           if ("broken_link" in attrs.get('class')):
-               log.debug(LOG_CHECK," Found a broken_link to %s, not considering further", attrs.get('href'))
-               return
+        ignoreclasses = ['broken_link','borked_link']
+        if tag == "a" and attrs.get('class') and ignoreclasses:
+            if any(item in ignoreclasses for item in attrs.get('class').split()):
+                log.debug(LOG_CHECK,"Found link classed \"%s\" to %s, not considering further", attrs.get('class'), attrs.get('href'))
```

Comment:
Style tip: space after a comma.

Style tip: instead of backslash-escaping double quotes inside double quotes, use single quotes for the string literal:

```suggestion
                log.debug(LOG_CHECK, 'Found link classed "%s" to %s, not considering further', attrs.get('class'), attrs.get('href'))
```